From 24946410d00c1c8979dfaf0c16abac0a39d52e28 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 30 Nov 2020 13:29:30 +0000
Subject: [PATCH 259/699] dt-bindings: ili9881c: add compatible string for new
 panels

This new panel uses the ILI9881C IC but needs an alternate
init sequence, and therefore requires a new compatible string.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../devicetree/bindings/display/panel/ilitek,ili9881c.yaml      | 2 ++
 1 file changed, 2 insertions(+)

--- a/Documentation/devicetree/bindings/display/panel/ilitek,ili9881c.yaml
+++ b/Documentation/devicetree/bindings/display/panel/ilitek,ili9881c.yaml
@@ -22,6 +22,8 @@ properties:
           - startek,kd050hdfia020
           - tdo,tl050hdv35
           - wanchanglong,w552946aba
+          - raspberrypi,dsi-5inch
+          - raspberrypi,dsi-7inch
       - const: ilitek,ili9881c
 
   reg:
